curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-asynchronous-programming-by-building-an-fcc-forum-leaderboard/645f7879ebbdb201892e55e1.md

```diff
@@ -26,7 +26,7 @@ assert(code.match(/const\s+userAvatarUrl\s*=/));
 You should assign a `ternary` operator to the `userAvatarUrl`. Your `ternary` should check if `avatar` starts with `/user_avatar/` and return `avatarUrl.concat(avatar)` or `avatar`.
 
 ```js
-assert(code.match(/const\s+userAvatarUrl\s*=\s*avatar\.startsWith\(\s*('|")\/user_avatar\/\1\s*\)\s*\?\s*avatarUrl\.concat\(\s*avatar\s*\)\s*:\s*avatar\s*/));
+assert(code.match(/const\s+userAvatarUrl\s*=\s*(\/\^user_avatar\/\.test\(avatar\)\s*\?\s*avatarUrl\.concat\(avatar\)\s*:\s*avatar;?|avatar\.startsWith\(\s*('|")\/user_avatar\/\2\s*\)\s*\?\s*avatarUrl\.concat\(\s*avatar\s*\)\s*:\s*avatar\s*)/));
```

Comment:
```suggestion
assert(code.match(/const\s+userAvatarUrl\s*=\s*(?:\/\^user_avatar\/\.test\(avatar\)|avatar\.startsWith\(\s*('|")\/user_avatar\/\1\s*\)\s*)\s*\?\s*avatarUrl\.concat\(\s*avatar\s*\)\s*:\s*avatar\s*;?/));
```
This will be a bit simpler. Only the `/^user_avatar/.test(avatar)` and `avatar.startsWith('user_avatar')` parts are differing, only they are included in the alternatives.

Reply:
Hey @jayanth920 ! 

It looks like we still have this unresolved comment here.
If you have any questions about this, please let us know so it can be resolved 👍